.devcontainer/Dockerfile

```diff
@@ -2,4 +2,4 @@ FROM mcr.microsoft.com/vscode/devcontainers/java:11
 
 RUN apt-key adv --refresh-keys --keyserver keyserver.ubuntu.com\
   && apt-get update && export DEBIAN_FRONTEND=noninteractive \
-  && apt-get -y install --no-install-recommends leiningen
+  && apt-get -y install --no-install-recommends leiningen nodejs yarn
```

Comment:
Much closer now but when I try and run `yarn dev` it fails with the following message. ```error @percy/cli@1.0.0-beta.60: The engine "node" is incompatible with this module. Expected version ">=12". Got "10.24.0"```

Running `node -v` I do indeed get `10.24.0` so my guess is that the `nodejs` default being installed is too low a version.

Reply:
Oops, now I tweaked it to install v14 instead.